governance.md

```diff
@@ -88,6 +89,7 @@ Current TOC members:
 * Adam Jordens (Netflix)
 * Rob Zienert (Netflix)
 * Eric Zimanyi (Google)
+* Michael Plump (Google)
```

Comment:
Is the intention to also vote on this as part of this PR?

Reply:
@demobox I believe this was voted on at yesterday's SC meeting, according to #32

I didn't see this PR, so I added myself to this page in #36 (since I was already added to a different page in #34). Sorry! :grin: